service-front/module/Application/src/Model/Service/Signatures/DateCheck.php

```diff
@@ -146,13 +146,8 @@ public static function checkDates(
 
                 // Donor must be first
                 if ($donor > $timestamp) {
-                    $message = "${donorText} must be the first person to sign the LPA. ";
```

Comment:
$isDraft is always false, otherwise you're not in this block at all; therefore the `if ($isDraft)` condition never passes, and the code inside is dead.